docs/hsi.xml

```diff
@@ -393,6 +393,22 @@
       </note>
     </refsect3>
 
+    <refsect3 id="org.fwupd.hsi.Spi.Descriptor">
+      <title>Read-only SPI Descriptor</title>
+      <para>
+        The SPI descriptor must always be read only from all other regions.
+        Additionally the FLOCKDN register must be set to prevent configuration
```

Comment:
I think you need something like ""Additionally on Intel architectures..."

Reply:
Agree, thanks.